FieldSupplyAIDriver.lua

```diff
@@ -21,90 +21,127 @@ Field Supply AI Driver to let fill tools with digestate or liquid manure on the
 Also known as mode 8
 ]]
 
---TODO: Have FieldSupplyAIDriver be derived from GrainTransportDriver and not FillableFieldworkAIDriver,
---		as there is no need for FieldworkAIDriver functions 
-
----@class FieldSupplyAIDriver : FillableFieldworkAIDriver
-FieldSupplyAIDriver = CpObject(FillableFieldworkAIDriver)
+---@class FieldSupplyAIDriver : GrainTransportAIDriver
+FieldSupplyAIDriver = CpObject(GrainTransportAIDriver)
 
```

Comment:
Changes the inheritance to GrainTransportAIDriver, as we don't need the field course and so on. 
Might have to heck if this change impactes autodrive setup.